in_toto/runlib.py

```diff
@@ -350,15 +350,29 @@ def execute_link(link_cmd_args, record_streams):
     - The return value of the executed command.
   """
   if record_streams:
-    return_code, stdout_str, stderr_str = \
-        securesystemslib.process.run_duplicate_streams(link_cmd_args)
+    if (quiet == False): #record_streams true, quiet false
+      return_code, stdout_str, stderr_str = \
+      securesystemslib.process.run_duplicate_streams(link_cmd_args)
+    else: #record_streams true, quiet true
+     process = securesystemslib.process.run(link_cmd_args, check=False,
+     stdout=securesystemslib.process.PIPE,
```

Comment:
```suggestion
       stdout=securesystemslib.process.PIPE,
```

Reply:
Oh, YES! I was looking for the correct way to do this one. I wasn't quite sure about this -- thank you!